7.application_configuration.md

```diff
@@ -70,70 +68,62 @@ The specification defines three major things: Parameters that can be overridden
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The parameter's name. Must be unique per configuration. |
-| `value` | `string` | Y | | The scalar value. |
-
-A `name` is any Unicode letter or numeric character, `_`, `-`, and `.`. A `value` is any sequence of printable Unicode characters.
+| `scope` | [`ScopeRef`](#scopeRef) | Y | | The reference information of the Scope. |
 
-The function `[fromVariable(VARNAME)]` (where `VARNAME` corresponds to the `name` field in a variable definition) may be used in predefined locations (parameter values and properties objects) to access the `value` of a parameter.
+### ScopeRef
 
-### Scope
-The scope section defines application scopes that will be created with this application configuration.
+The scopeRef section will index a scope instance by `apiVersion`, `kind` and `name`.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `name` | `string` | Y | | The name of the application scope. Must be unique to the deployment environment. |
-| `type` | `string` | Y | | The fully-qualified GROUP/VERSION.KIND name of the application scope. |
-| `properties` | [`Properties`](#properties) | N | | Overrides of properties that are exposed by this scope. |
+| `apiVersion` | `string` | Y | | The apiVersion of the Scope. |
+| `kind` | `string` | Y | | The kind of the Scope. |
+| `name` | `string` | Y | | The name of the Scope. |
+
+> The name field is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
 
-> The name field must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
 
 ### Component
 This section defines the instances of components to create with this application configuration.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
 | `componentName` | `string` | Y | | The name of the component to create an instance of. |
-| `instanceName` | `string` | Y | | The name of the instance of this component. |
 | `parameterValues` | [`[]ParameterValue`](#parameterValue) | N | | Overrides of parameters that are exposed by the application scope type defined in `type`. |
 | `traits` | [`[]Trait`](#trait) | N | | Specifies the traits to attach to this component instance. |
-| `applicationScopes` | `[]string` | N | | Specifies the application scopes that the component should be deployed into. Each item is a name referenced to the pre-defined Scopes. |
+| `scopes` | [`[]string`](#scope) | N | | Specifies the scopes' name defined in [spec.scopes](#scope) that the component should be deployed into. |
```

Comment:
The data type is right (list of strings) but the name of the field should be `scopeRefs` if we're following Kubernetes conventions (https://github.com/kubernetes/community/blob/96203/contributors/devel/sig-architecture/api-conventions.md#naming-conventions)